Sources/MintKit/MintKit.swift

```diff
@@ -103,11 +103,15 @@ public struct Mint {
         try shellOut(to: "git fetch --tags", at: package.checkoutPath.string)
 
         if package.version.isEmpty {
-            let tag = try shellOut(to: "git describe --abbrev=0 --tags", at: package.checkoutPath.string)
-            if !tag.isEmpty {
-                package.version = tag
-                print("🌱  Using latest tag \(tag.quoted)")
-            } else {
+            do {
+                // This will exit with a non-zero status code when there are no tags
+                let tag = try shellOut(to: "git describe --abbrev=0 --tags", at:  package.checkoutPath.string)
+
+                if !tag.isEmpty {
```

Comment:
This check probably isn't necessary anymore then. If tag was empty here it also wouldn't resolve to anything